spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -213,6 +213,7 @@
 "GNU GENERAL PUBLIC LICENSE Version 2, June 1991": GPL-2.0-only
 "GNU GPL v2": GPL-2.0-only
 "GNU General Lesser Public License (LGPL) version 2.1": LGPL-2.1-only
+"GNU General Lesser Public License (LGPL) version 3.0": LGPL-3.0
```

Comment:
Actually, this shouldn't have passed our tests as `LGPL-3.0` is a deprecated id; please use `LGPL-3.0-only` or `LGPL-3.0-or-later` instead. I'll look at improving the tests.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/3360.

Reply:
Done